web/src/components/project/component/List.js

```diff
@@ -44,7 +40,7 @@ class List extends React.Component {
     const columns = [
       {
         title: intl.get('name'),
-        dataIndex: 'metadata.name',
+        dataIndex: 'metadata.annotations["cyclone.io/alias"]',
```

Comment:
if get undefined , loadash more better?

Reply:
Here `metadata.annotations["cyclone.io/alias"]` is key path in the object, not retrieve data here.